### PR TITLE
Added icon that links to Jiya's portfolio

### LINKED ISSUE #1: Add Jiya's image icon
Need to add an image icon to the homepage that leads to Jiya's portfolio page.

### PR DESCRIPTION
**Pull request by jiyapatel4**
Upon clicking Jiya's image in the menu bar, you are sent to her resume/portfolio page. 

Closes #1

### CODE REVIEW

File: app/__init__.py
Comment (chizy7) [inline app/__init__.py:1]: Hi @jiyapatel4,

Great job setting up the new route and keeping your code well-commented. Here are a few suggestions to consider:

- For the '/jiya_portfolio' route, consider if there's any relevant context that should be passed into the 'render_template' function.
- Please add error handling to ensure the application can gracefully handle situations where the 'jiya_portfolio.html' template is missing or fails to load.

Could you also confirm that:
- The new route '/jiya_portfolio' works as expected?
- The 'jiya_portfolio.html' page loads correctly and displays the intended content?

Looking forward to your updates.

Thanks!

File: app/templates/index.html
Comment (chizy7) [inline app/templates/index.html:1]: Thanks for your work on this. The link to Jiya's portfolio page is a good addition and its implementation is correct.

Here are a few suggestions for improvements:

- For better accessibility, consider adding an `alt` attribute to the `img` tag.

Could you also confirm that:
- Clicking Jiya's image correctly redirects to Jiya's portfolio page?
- The 'headshot.jpg' image is loading correctly?

File: app/templates/jiya_portfolio.html
Comment (chizy7) [inline app/templates/jiya_portfolio.html:1]: Here are my comments:

- Consider adding more structure to this page for the different portfolio sections (such as about, skills, projects, etc.).
- There seems to be a typo in the stylesheet link with the 'lang' attribute, which isn't standard practice.

**Review by chizy7 — CHANGES_REQUESTED** [review]
Great job on a great start!

File: app/__init__.py
Comment (jiyapatel4) [reply app/__init__.py:1]: Hey, @chizy7 all your feedback is very thorough, thank you!

Updates I made:
- I passed in a title and url variable to `render_template()`.
- I added error handling for 404 errors in `__init__.py`, but I'm not sure how to cover all possible errors.
-  Yes, the route `/jiya_portfolio` works and `jiya_portfolio.html` renders correctly.

Any advice on the error handling?

File: app/templates/index.html
Comment (jiyapatel4) [reply app/templates/index.html:1]: Updates I made:
- Good call, I added the `alt` attributes.
- Yes, clicking Jiya's Image correctly redirects to Jiya's portfolio. 
- Yes, headshot.jpg loads correctly.

File: app/templates/jiya_portfolio.html
Comment (jiyapatel4) [reply app/templates/jiya_portfolio.html:1]: - I plan on adding the sections in a separate branch and creating a new PR.
- I believe the stylesheet link with the 'lang' attribute is from the given template, should I remove it?

File: app/templates/index.html
Comment (chizy7) [reply app/templates/index.html:1]: Awesome, thanks for the clarification.

File: app/__init__.py
Comment (chizy7) [reply app/__init__.py:1]: I'm glad to hear that the updates were helpful and that the 'jiya_portfolio' route and template are working as expected.

Note: I will approve the PR. Also, to answer your question on error handling:

For error handling in Flask, you can implement a few strategies to cover most scenarios:
1. HTTP Errors: Flask allows you to handle specific HTTP error codes using the `@app.errorhandler(error_code)` decorator. You've already implemented this for 404 errors, which is great. You might want to add error handlers for other common HTTP error codes such as 400 (Bad Request), 403, 405, 500codes.

```@app.errorhandler(500)
def internal_server_error(error):
    return render_template('500.html'), 500
```
2. Exceptions: ou can catch exceptions that might occur during the execution of your code. 
3. Teardown Callbacks.

Implementing these strategies can provide a better user experience when things go wrong and also make it easier to debug issues.

Let me know if you have any other questions.

**Review by chizy7 — APPROVED** [review]
Looks good

**Pull request merged**